### DISCUSSION TITLE
onhover show larger image of a thumbnail as a tooltip popup

### DISCUSSION THREAD

**Discussion by anjanesh** [tailwindlabs/tailwindcss#12799] [Help]
How do I make a mouseover to show a larger image of a thumbnail ?

What I did years ago was using jQuery :

![image](https://github.com/tailwindlabs/tailwindcss/assets/85973/b5c58953-a070-4e51-996f-8e3cb8bc0748)

**Comment by wongjn** [comment]
Here's a basic solution: https://play.tailwindcss.com/xGtL4nBI47

**Reply by anjanesh** [reply]
I found a solution at Flowbite which is what I was searching for using plain TailWindCSS. I guess JavaScript is required for this.
https://flowbite.com/docs/components/popover/
